sur/library.hoon

```diff
@@ -0,0 +1,50 @@
+/-  *resource, store=graph-store
+|%
++$  comment  tape
++$  book
+  $:
+    title=tape
+    isbn=tape
+    :: TODO fulltext=tape ? 
+  ==
+::
+::  TODO better / clearer name for prim
++$  prim     (jug resource atom)  :: for a given resource, what indexes does the reader want to hear about?
++$  readers  (map ship prim)      :: given a ship, what are the resources and books that it cares about?
+::  TODO most things that take a rid can theoretically be just a name=@tas because entity=@p is always gonna be the owner of the proxy
```

Comment:
yes remove the redundancy

Reply:
https://github.com/ynx0/library/pull/3/commits/243ca90a3e12420a464436c3a35012097c670d70